### DISCUSSION TITLE
Visualize execution history in graphical-editor/viewer

### DISCUSSION THREAD

**Discussion by franzlst** [DLR-RM/RAFCON#909] [Ideas]
The last x states of the execution history could be visualized in another color to make it easier for the user to track the execution.

The issue is a duplication of mantis issue https://rmintra01.robotic.dlr.de/mantis/view.php?id=2224.

Originally created by @Rbelder  at 2016-03-02 13:49:19+00:00 (moved from RMC internal repository)

**Comment by JohannesErnst** [comment]
Or maybe give states that were aborted a different color in the execution history so they can easily be spotted when debugging
